conf/abims.config

```
@@ -0,0 +1,24 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'The ABiMS cluster profile'
+  config_profile_contact = 'Gildas Le Corguillé (@lecorguille)'
+  config_profile_url = 'https://abims.sb-roscoff.fr'
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
```

```suggestion
  autoMounts = false
```
According to your comment, you should change that ;-)

---

Oh, it's easy!
Do you have a clue if it's a special setting within our infrastructure or related to our software versions or my usage or ...?
Otherwise, 🐑 mode enable!

---

I'd say settings on your cluster.

---

Copy that!